model/project_selector.go

```diff
@@ -222,8 +236,11 @@ func (tse *tagSelectorEvaluator) evalCriterion(sc selectCriterion) ([]string, er
 	case sc.tagged && sc.negated: // everything *but* a tag
 		items := tse.byTag[sc.name]
 		if len(items) == 0 {
-			// we want to treat this as an error for better usability
-			return nil, errors.Errorf("nothing has the tag '%v'", sc.name)
+			if !tse.ignoreUnmatchedTags {
+				// we want to treat this as an error for better usability
```

Comment:
Can we remove this comment + the others? It seems kind of weird to say it's better usability when it's now a conditional error that seems less easily usable.